public/index.html

```diff
@@ -46,17 +46,16 @@ <h3 id = "date"></h3>
             <input type="radio" value="n" name="n-s">Arctic
             <input type="radio" value="s" name="n-s">Antarctic
             <br>
-            Starting Date (1979-2018): <input type = "date" name = "sDate">
+            Starting Date (1979-2020): <input type = "date" name = "sDate">
```

Comment:
Ideally populate the end date with `moment`, to get the current year.

Reply:
Set to 1 day previous in daily mode and 1 month previous in monthly mode.